### ISSUE TITLE
[Accessibility] "Please wait a moment..." is low contrast

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [ampproject/amp.dev#4488]
Environment:
Any OS/browser

Steps to reproduce:
1. Open https://amp-dev-staging.appspot.com/page-experience/
2. Enter a valid URL and press Analyze
3. Observe the text that says:
>Please wait a moment
>We are currently analyzing your page . . .
>16 out of 28 checks are completed

The current contrast ratio of this text against the background is 1.61:1
Text color: #CCCCCC
Background color: #FFFFFF

The text/contrast ratio should be at least 4.5:1 for most elements. See [WCAG 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum) and [Understanding Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

![image](https://user-images.githubusercontent.com/30604285/92812617-7735ec80-f374-11ea-9843-b29913d2c91d.png)
![image](https://user-images.githubusercontent.com/30604285/92812548-51a8e300-f374-11ea-9246-549ad0cb2209.png)
